add_gist.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rubygems'
+require 'net/http'
+require 'uri'
+require 'json'
+require 'OpenSSL'
+require 'find'
+
+class PathNotFoundError < StandardError; end
+
+def read_files(pwd = Dir.pwd)
+  files = {}
+  begin
+    unless File.exist?(pwd)
+      raise PathNotFoundError, 'Path to file or directory not found'
+    end
+    Find.find(pwd) do |path|
+      if File.file?(path)
+        files.merge!(File.basename(path) => { 'content': File.open(path, 'r+').read })
+      end
+    end
+  rescue PathNotFoundError => e
+    puts e.message
+  end
+  files
+end
+
+def read_access_token
+  file = File.open('./accessToken.env', 'r+')
```

Comment:
I would like you to try the dotenv gem https://github.com/bkeepers/dotenv we talked about